Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -906,6 +906,48 @@ public static T TryFindParent<T>(DependencyObject child)
             else
                 return TryFindParent<T>(parentObject);
         }
+
+        /// <summary>
+        /// This function is used to replace the password value from the string with *****
+        /// </summary>
+        /// <param name="dataString">The string argument which has password value</param>
+        /// <returns></returns>
+        public static string HidePasswordFromString(string dataString)
+        {
+            string passwordValue = dataString.Replace(" ", "");//remove spaces
+            string passwordString = string.Empty;
+            //Matching string
+            if (dataString.ToLower().Contains("pwd="))
+            {
+                passwordString = "pwd=";
+            }
+            else if (dataString.ToLower().Contains("password="))
+            {
+                passwordString = "password=";
+            }
+            else
+            {
```

Comment:
please add unit tests for all types on connection strings password/pwd combinations

Reply:
Done